modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the acquia_cms_common module.
+ */
+
+use Drupal\Core\Config\FileStorage;
+
+/**
+ * Add new text format 'Full HTML'.
+ */
+function acquia_cms_common_update_8001() {
+  // Do not override existing full_html, if already provide.
+  $config_full_html = \Drupal::config('filter.format.full_html')->get('name');
+  ;
```

Comment:
@chandan-singh7929 Can you remove this extra line ?

Reply:
Done.